overlord/snapstate/target.go

```diff
@@ -312,9 +312,7 @@ func (s *storeInstallGoal) toInstall(ctx context.Context, st *state.State, opts
 			channel = sn.RevOpts.Channel
 		default:
 			// this should only ever happen if the caller requested a specific
-			// revision to be installed (without specifying a channel), but the
-			// store didn't return a redirect channel. this would happen if the
-			// snap doens't have a default track
+			// revision to be installed (without specifying a channel)
 			channel = "stable"
```

Comment:
do we know though what happens when we pass just stable on a refresh? refreshes don't tend to use default tracks

Reply:
That shouldn't be a concern, as we don't actually track just "stable". I've updated the comment here to clarify that a bit.